cime_config/allactive/config_pesall.xml

```diff
@@ -1482,6 +1482,31 @@
           <nthrds_cpl>1</nthrds_cpl>
         </nthrds>
       </pes>
+      <pes compset=".*SCREAM.*" pesize="any">
+        <comment>summit|ascent: BGC compset on ne30np4.pg2 grid</comment>
```

Comment:
Update comment and include `SCREAM`: this comment shows up in env_mach_pes.xml and can be helpful.